stacks_queues/queue.py

```diff
@@ -10,52 +10,57 @@ class QueueEmptyException(Exception):
 class Queue:
 
     def __init__(self):
-        self.store = [None] * INITIAL_QUEUE_SIZE
-        self.buffer_size = INITIAL_QUEUE_SIZE
-        self.front = -1
-        self.rear = -1
-        self.size = 0
-      
+        self.store = []      
 
     def enqueue(self, element):
         """ Adds an element to the Queue
             Raises a QueueFullException if all elements
             In the store are occupied
             returns None
         """
-        pass
+        if len(self.store) == INITIAL_QUEUE_SIZE:
+            raise QueueFullException
+        
+        self.store.append(element)
```

Comment:
⚠️ This isn't done via a [circular buffer](https://www.programiz.com/dsa/circular-queue).  

So it's not really fulfilling the assignment.